From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
Date: Wed, 1 Nov 2017 13:04:00 +0300
Subject: [ARCHEOLOGY] Lower default CPU and DRAM frequencies for H5 and A64
 devices

> X-Git-Archeology: - Revision 46e082773acb379ce3b3aa0c8fec42bb48750830: https://github.com/armbian/build/commit/46e082773acb379ce3b3aa0c8fec42bb48750830
> X-Git-Archeology:   Date: Wed, 01 Nov 2017 13:04:00 +0300
> X-Git-Archeology:   From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
> X-Git-Archeology:   Subject: Lower default CPU and DRAM frequencies for H5 and A64 devices
> X-Git-Archeology:
> X-Git-Archeology: - Revision 9363e3ec4d3a3234c5bdab6b7a1e3fd606b68102: https://github.com/armbian/build/commit/9363e3ec4d3a3234c5bdab6b7a1e3fd606b68102
> X-Git-Archeology:   Date: Mon, 14 Jan 2019 16:52:22 -0500
> X-Git-Archeology:   From: Martin Ayotte <martinayotte@yahoo.ca>
> X-Git-Archeology:   Subject: u-boot v2018.11 migration + tons of patches touchups
> X-Git-Archeology:
> X-Git-Archeology: - Revision f6a09def35e647a5442e0f92f399485be29f19a9: https://github.com/armbian/build/commit/f6a09def35e647a5442e0f92f399485be29f19a9
> X-Git-Archeology:   Date: Thu, 10 Nov 2022 21:49:36 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Moving patches to per board and removing obsolete (#4409)
> X-Git-Archeology:
---
 arch/arm/mach-sunxi/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-sunxi/Kconfig b/arch/arm/mach-sunxi/Kconfig
index 6417aee944..08cfc581cb 100644
--- a/arch/arm/mach-sunxi/Kconfig
+++ b/arch/arm/mach-sunxi/Kconfig
@@ -467,7 +467,7 @@ config DRAM_CLK
 	default 312 if MACH_SUN6I || MACH_SUN8I
 	default 360 if MACH_SUN4I || MACH_SUN5I || MACH_SUN7I || \
 		       MACH_SUN8I_V3S
-	default 672 if MACH_SUN50I
+	default 648 if MACH_SUN50I || MACH_SUN50I_H5
 	default 744 if MACH_SUN50I_H6
 	default 720 if MACH_SUN50I_H616
 	---help---
-- 
Armbian

